services/filemanager/filemanager.go

```diff
@@ -56,11 +56,6 @@ func init() {
 	DefaultFileManagerFactory = &FileManagerFactoryT{}
 }
 
-// Deprecated: Use an instance of FileManagerFactory instead
-func New(settings *SettingsT) (FileManager, error) {
-	return DefaultFileManagerFactory.New(settings)
-}
```

Comment:
@lvrach we're using this on `enterprise` though, what do you suggest?

```
enterprise/migrator/migrator.go:126:25: New not declared by package filemanager (typecheck)
	fm, err := filemanager.New(&settings)
	                       ^
```

Reply:
Yeah, we can create a PR for enterprise as well.

Reply:
Done here: https://github.com/rudderlabs/rudder-server-enterprise/pull/91